.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
Side-note: `.DS_Store` is a macOS specific thing. For OS / editor specific things global gitignore (https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/) is often a nice solution.

Suggestions on how we treat such in the future? Add those to repo gitignore or request contributors to setup their own gitignore?

Reply:
I wouldn't overthink this. You're ofc right with your observation and in this phase I'm mainly focused on making it easy for folks to contribute, not establish too many barriers. If you feel strongly about it, I suggest you create an issue and let's discuss it there?

Reply:
No need, just wanted to raise it in case it was an oversight. It totally doesn't matter in this stage of the project :+1: